.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,19 +13,55 @@ For questions, the best place to get answers is on our [discussion forum](https:
 
 Please note: We take Boundary's security and our users' trust very seriously. If you believe you have found a security issue in Boundary, please responsibly disclose by contacting us at security@hashicorp.com. Our PGP key is available at [our security page](https://www.hashicorp.com/security/).
```

Comment:
nit: Please note: We take Boundary's security and our users' trust very seriously. If you believe you have found a security issue in Boundary, please responsibly disclose `this` by contacting us at security@hashicorp.com. Our PGP key is available `on` [our security page](https://www.hashicorp.com/security/).